src/template/wasm2c.declarations.c

```diff
@@ -47,10 +47,18 @@ static inline bool func_types_eq(const wasm_rt_func_type_t a,
 #define MEMCHECK(mem, a, t) RANGE_CHECK(mem, a, sizeof(t))
 #endif
 
-#ifdef __GNUC__
-#define wasm_asm __asm__
+#if !defined(WASM2C_DISABLED_STRICT_TRAPPING) && defined(__GNUC__)
```

Comment:
Given that our fallback is still going to be "don't force the read" (instead of "bomb out with an error"), can we avoid adding the `WASM2C_DISABLED_STRICT_TRAPPING` macro?

Reply:
Fixed!